integrationtest/src/androidTest/java/com/swiftkey/cornedbeef/CoachMarkTestCase.java

```diff
@@ -113,6 +113,17 @@ public void run() {
      * Test that the popup disappears after the anchor view detaches if specified in the builder
      */
     public void testDismissAfterAnchorDetach() throws InterruptedException {
+        testSetDismissOnAnchorDetach(true);
+        testSetDismissOnAnchorDetach(false);
+    }
+
+    /**
+     * Helper method used in testDismissAfterAnchorDetach
+     *
+     * @param shouldAutoDismissOnDetach whether we should auto dismiss the coach mark
+     * @throws InterruptedException
+     */
+    private void testSetDismissOnAnchorDetach(boolean shouldAutoDismissOnDetach) throws InterruptedException {
```

Comment:
Can you call this 'checkSetDismissOnAnchorDetach' rather than 'test' to distinguish it from the actual test